### PR TITLE
Make console.log stricter, allowing only info, warn, and error, add a…

### PR DESCRIPTION
**Pull request by jcalleja-hubspot**
Make console.log stricter, allowing only info, warn, and error, add anpmrc sample, and bump the major version as it's a breaking change

### CODE REVIEW

File: .npmrc.sample
Comment (darkmavis1980) [inline .npmrc.sample:1]: Actually, we don't need this, we want to use the `npm version <major|minor|patch>` here, as we don't have Jenkins doing the tagging for us

File: package.json
Comment (darkmavis1980) [inline package.json:3]: Maybe we should just do a minor version, we are still in version `0` so breaking changes are ok here

Comment (jcalleja-hubspot) [reply package.json]: Makes sense 🙌, I've just changed that to be a minor version in the last commit

File: .npmrc.sample
Comment (jcalleja-hubspot) [reply .npmrc.sample]: Makes sense 🙌, I've just removed that in the last commit

**Review by darkmavis1980 — APPROVED** [review]

**Comment by jcalleja-hubspot** [comment]
Hey 👋, just a heads up, it seems like we need an approval from a maintainer:
Screenshot 2024-12-11 at 9 59 04 AM
But the [codeowners](https://github.com/HubSpotWebTeam/wt-eslint-node/blob/main/.github/CODEOWNERS) seems to have an error:
Screenshot 2024-12-11 at 10 00 18 AM

Somebody with admin write access might need to modify that, unless I'm missing something 😅

**Pull request merged**
